branding/spacewalk-branding.spec

```diff
@@ -24,8 +24,10 @@
 %global wwwdocroot %{_var}/www/html
 %else
 %if 0%{?suse_version}
-%global tomcat_path /srv/tomcat
-%global wwwdocroot /srv/www/htdocs
+%global susemanager_shared_path /usr/share/susemanager
```

Comment:
Do we  really want to use `susemanager` as name for the  folder? In this way we are  going to hardcode the same name  also in uyuni. If we really want to have the product name as name  of the folder, what about  having a conditional that set `uyuni` or `susemanager` according to the product?

Reply:
Why not even set `uyuni` for all?

Reply:
so do we want to replace `/usr/share/susemanager` to `/usr/share/uyuni` (also for the files that were there before my refactoring)?

Reply:
This is a bigger move and we should probably not consider it for this PR, but IMHO this is something we would need to think about at some point.

Reply:
> please not change it twice in different official updates. This require again migration effort
I agree. I think it could be better to have the naming fixed now and not in two  steps

Reply:
Agree, we should define the name once and change it for good

Reply:
What about doing that change in a separate PR, but merge the two in the same release. We can even merge them together, but that would make the whole change easier to review... I'ld bet this would add quite some diffs more

Reply:
For me this would be ok. I only want to prevent, that we do an uyuni release with this name and have to change it later to another including a migration script

Reply:
My 2c is to keep susemanager everywhere. Even though SUSE Manager is not the best name, it is established name which our end goal is to promote the most as that is what is paying our bills at the end.

In my opinion we should rather focus on renaming all spacewalk things to susemanager then susemanager to uyuni.

Reply:
my 2 cents... 
I would call it `uyuni` and have it released with the next 2023.10. it's ok to  do it  in 2 PRs, but  please, don't split this change in two releases

Reply:
after so many cents we may manage to build something like this one:
![image](https://github.com/uyuni-project/uyuni/assets/397931/9839631b-61d1-4a2e-b2f3-fb4367cf303d)